dashboard/app/controllers/projects_controller.rb

```diff
@@ -304,6 +305,7 @@ def show
       hide_source: sharing,
       share: sharing,
       iframe_embed: iframe_embed,
+      iframe_embed_app_and_code: iframe_embed_app_and_code,
       project_type: params[:key]
     )
     # for sharing pages, the app will display the footer inside the playspace instead
```

Comment:
Get rid of the 'no_footer' variable, just assign no_footer directly as sharing || iframe_embed_app_and_code on line 320. This variable just takes up space

Reply:
My comment was a reminder! I was doing a self-review as part of the code review